pieces/scenario/atlaskit-drag-and-drop/card.tsx

```diff
@@ -134,6 +141,19 @@ export const Card = memo(function Card({ item }: { item: Item }) {
       <DragIcon state={state} />
       <CardText state={state} />
       <DropIndicator edge={closestEdge} gap={'var(--card-gap)'} />
+      <div css={controlStyles}>
+        <MenuButton label={`controls for card ${itemId}`}>
+          <MenuItem>Edit</MenuItem>
+          <MenuItem>Share</MenuItem>
+          <MenuItem>Move up</MenuItem>
+          <MenuItem>Move down</MenuItem>
+          {['A', 'B', 'C']
```

Comment:
Nice! Not overly complicated and gets the job done 👍 

A thought: could we pass all column ids as an array to each card? That way we don't need to hard code them

Up to you

Reply:
I agree not hard coding is better. I originally did it as a quick solution, but have revised it now to pass them down :)